### DISCUSSION TITLE
GTD: How can I get started?

### DISCUSSION THREAD

**Discussion by weberam2** [nvim-neorg/neorg#1369] [Q&A]
Sorry if this isn't implemented yet

But I saw this: https://github.com/nvim-neorg/neorg/discussions/67

And was hoping to start trying it, but I get the error: "module 'neorg.modules.core.gtd.base.module' not found: No LuaRocks module..."

Any way to try it out?

**Comment by max397574** [comment]
it isn't implemented yet
it was in the past but got removed and now there are new things being worked on
but atm there is no gtd functionality in neorg

**Reply by weberam2** [reply]
Sounds good :)

**Reply by GitMurf** [reply]
@max397574 is GTD on hold or is it being abandoned? is there plans for a replacement / new direction? Thanks!

**Reply by max397574** [reply]
still same as above
